### PR TITLE
eventbrite widget loading is now more correct

### PR DESCRIPTION
**Pull request by jamesgorrie**
There is a very small edge case that a user might see the button before the widget has loaded.
No we have feedback saying we're in the process of loading.
Also makes the button accessible as a link again...

eventbriteloading

### CODE REVIEW

File: common/views/components/BasePage/EventPage.js [hunk 1]
```diff
@@ -169,6 +169,9 @@ const EventPage = ({ event }: Props) => {
     topLink={{text: 'Events', url: '/events'}}
   />);
 
+  const ticketButtonText = 'Check for tickets';
+  const ticketButtonLoadingText = 'Checking…';
```
Comment (davidpmccormick) [inline common/views/components/BasePage/EventPage.js]: Not convinced 'Checking…' is the right messaging for this (i.e. waiting for the iframe to load). Maybe, 'Wait…' or 'Loading…' or 'Give us a second…'?

Comment (jamesgorrie) [reply common/views/components/BasePage/EventPage.js]: ping @Heesoomoon

Comment (jamesgorrie) [reply common/views/components/BasePage/EventPage.js]: Could we have consistent messaging for this? In that case maybe `Loading…` works.

Comment (jamesgorrie) [reply common/views/components/BasePage/EventPage.js]: CHanged.

File: common/views/components/BasePage/EventPage.js [hunk 2]
```diff
@@ -247,14 +251,22 @@ const EventPage = ({ event }: Props) => {
                         (function() {
                           var iframe = document.getElementById('eventbrite-widget-${event.eventbriteId || ''}');
                           var showWidget = document.getElementById('eventbrite-show-widget-${event.eventbriteId || ''}');
+                          showWidget.classList.add('disabled');
```
Comment (davidpmccormick) [inline common/views/components/BasePage/EventPage.js]: is `disabled` a class of ours? I don't see it in our repo

Comment (jamesgorrie) [reply common/views/components/BasePage/EventPage.js]: https://github.com/wellcometrust/wellcomecollection.org/pull/3173/files#diff-db894dd50299dc4d3525dfb5690f5369R13

**Pull request merged**